metrics/README.md

```diff
@@ -54,9 +41,6 @@ jqmeasurements: |
 * flakes - find the flakiest jobs this week (and the flakiest tests in each job).
     - [Config](configs/flakes-config.yaml)
     - [flakes-latest.json](http://storage.googleapis.com/k8s-metrics/flakes-latest.json)
-* flakes-daily - find flakes from the previous day. Similar to `flakes`, but creates more granular results for display in Velodrome.
```

Comment:
flakes daily is still a thing, just not in velodrome
this is causing the tests to fail

Reply:
fixedd